application/event_processor/change_event_validation.py

```diff
@@ -10,7 +10,8 @@
 
 logger = Logger(child=True)
 
-ODSCODE_LENGTH = 5
+PHARMACY_ODSCODE_LENGTH = 5
+# DENTIST_ODSCODE_LENGTH = 6
```

Comment:
Is this variable required? as it is comment out. It should either be used or removed